app/views/RoomView/RightButtons.tsx

```diff
@@ -52,9 +55,12 @@ interface IRigthButtonsState {
 	tunreadGroup: string[];
 }
 
+const notificationIsEnabled = true;
```

Comment:
Why is this const fixed?

Reply:
to test the layout and create a variable with a name as expected

Reply:
Will this be removed then?